src/python/nispor/state.py

```diff
@@ -37,5 +37,5 @@ def routes(self):
         return self._routes
 
 
-def get_state():
-    return NisporState(json.loads(get_state_json()))
+def retrieve():
```

Comment:
How about we create a python struct `NetState` to be 1:1 matching with rust one?

Reply:
How about place it as static method as `NisporNetState.retrieve()`?

Reply:
The function `retrieve()` is still as module level rather than at `NisporNetState` class level.

Reply:
Sorry forgot, now it should be ok.